mono/metadata/appdomain.c

```diff
@@ -1762,13 +1762,13 @@ mono_make_shadow_copy (const char *filename, MonoError *oerror)
 	sibling_target_len = strlen (sibling_target);
 	
 	copy_result = shadow_copy_sibling (sibling_source, sibling_source_len, ".mdb", sibling_target, sibling_target_len, 7);
-	if (copy_result == TRUE)
```

Comment:
There's a similar comparison to `FALSE` further up that you may want to change as well.